images/win/Windows2019-Readme.md

```diff
@@ -18,9 +16,9 @@
 - Bash 4.4.23(1)-release
 - Go 1.15.15
 - Julia 1.6.4
-- Kotlin 1.6.0
+- Kotlin 1.5.32
```

Comment:
Version downgrade is related to an API response (https://api.github.com/repos/JetBrains/kotlin/releases/latest) which is used for Kotlin installation. Probably it's expected.

Reply:
This is not expected behavior that should be fixed in the next images rollout